src/code_0800A91C.c

```diff
@@ -0,0 +1,23 @@
+#include "global.h"
+#include "functions.h"
+
+// TODO: define file boundaries
```

Comment:
Is there a reason for you to put the function in a separate file, rather than keeping it in `code_0800A868.c`?

Reply:
That's what I was going to do originally but then it turns out there's already an existing declaration in `functions.h`, and so after consulting @Cleverking2003 I was told to move it to it's own file. (and leave the declaration in `functions.h`)

Reply:
It seems like I confused you a bit. I should have said you should have left definition in the file where it was before and not split it.